deploy-cropper.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+####################################################
+# Deploy the cropper
+#
+# Requires MCD environment variables to be in scope https://changelog.makerdao.com/releases/mainnet/active/contracts.json
+#
+# Usage: ./deploy-cropper.sh
+####################################################
+
+echo "Deploying contracts..."
+
+CROPPER=$(dapp create Cropper)
+CROPPER_IMP=$(dapp create CropperImp $MCD_VAT)
+
+echo "Set implementation..."
+
+seth send $CROPPER 'setImplementation(address)' $CROPPER_IMP
+
+echo "Set permissions..."
+
+seth send $CROPPER 'rely(address)' $MCD_PAUSE_PROXY
```

Comment:
Still just glancing but might be good to check here that MCD_PAUSE_PROXY is not empty (also a in the other script).
Just to avoid this failing, then denying and having to redeploy.

Reply:
or we can just use the chainlog to get the pause proxy addr